### DISCUSSION TITLE
[Idea] Add Hostname in System Informations

### DISCUSSION THREAD

**Discussion by Dreimus** [PrestaShop/PrestaShop#35040] [Ideas]
### The problem

When I have Prestashop in a load-balancing system, I can see the database name, but not the server hostname to identify on which server I am.

### The solution

Display hostname in SystemInformation

### Alternatives

Display a hash or another kind of crypted information associated to the hostname to identify the current server

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
> I'm willing to submit a Pull Request that implements this feature.

Go for it :)

**Comment by MatShir** [comment]
Go ! go ! :shipit:
